apps/site/layouts/layouts.module.css

```
@@ -61,6 +65,7 @@
       sticky
       bottom-0
       flex
+      h-[4rem]
```

```suggestion
      h-16
```

Similarly, you can use `h-16`